src/dsp_tools/commands/start_stack.py

```diff
@@ -94,15 +94,8 @@ def _get_url_prefix(self) -> str:
             return f"{url_prefix_base}/main/"
 
         config_file = importlib.resources.files("dsp_tools").joinpath("resources/start-stack/start-stack-config.yml")
-        if not config_file.is_file():
-            return f"{url_prefix_base}/main/"
-
-        with config_file.open("r", encoding="utf-8") as f:
-            try:
-                start_stack_config = yaml.safe_load(f)
-            except yaml.YAMLError:
-                start_stack_config = {}
-        commit_of_used_api_version = start_stack_config.get("DSP-API commit", "main")
+        start_stack_config = yaml.safe_load(config_file.read_bytes())
+        commit_of_used_api_version = start_stack_config["DSP-API commit"]
```

Comment:
Here, I did 2 things:
1. I simplified the with-clause to `config_file.read_bytes()`
2. I made it fail-fast. Before, the code was way too forgiving, so that it was possible to ship erroneous code to our users, because in our PRs, the tests erroneously passed. IMHO, if there is something wrong with the config file, we should get an error already in our PR, before we even ship the code.